README.md

```diff
@@ -67,6 +69,34 @@ with tracer.start_as_current_span('foo'):
             print(Context)
 ```
 
+### Metrics
+
+```python
+from opentelemetry import metrics
+from opentelemetry.sdk.metrics import Counter, Meter
+from opentelemetry.sdk.metrics.export import ConsoleMetricsExporter
+
+metrics.set_preferred_meter_implementation(lambda T: Meter())
```

Comment:
would be great if the SDK meter implementation didn't need this additional lambda. Feel a little weird that this pattern will have to be added in every codebase that wants to use the standard meter implementation.

Reply:
Right. I think as part of [#195], there will be an easier way to initialize these implementations (perhaps a default implementation is none is specified).